server/views/media_picker.py

```diff
@@ -34,16 +34,19 @@ def api_mediapicker_source_search():
     search_str = request.args['media_keyword']
     cleaned_search_str = None if search_str == '*' else search_str
     querying_all_media = False
+    link_id = request.args['link_id'] if 'link_id' in request.args else 0
```

Comment:
I  just realized that this could be simplified as: `request.args.get('link_id', 0)`

Reply:
that's default behavior of the get method? if it's there, use that value, otherwise default to 0? we use the if else everywhere

Reply:
Yes, that should be the case (see: https://docs.python.org/3/library/stdtypes.html#dict.get). I think we could use this syntax in a bunch of places!